### PR TITLE
translations: indigo-resolver/models.py

### PR DESCRIPTION
**Pull request by longhotsummer**

### CODE REVIEW

File: indigo_resolver/models.py [hunk 1]
```diff
@@ -19,14 +18,15 @@ class Authority(models.Model):
     """ Authority that knows how to resolve
     FRBR URIs into real-world URLs.
     """
-    name = models.CharField(max_length=255, unique=True, help_text="Descriptive name of this resolver")
-    url = models.URLField(help_text="Website for this authority (optional)", blank=True, null=True)
-    not_found_url = models.URLField(help_text="URL of a 404 page (optional)", null=True, blank=True)
-    slug = models.CharField(null=False, blank=False, validators=[validate_slug], max_length=50, unique=True)
-    priority = models.IntegerField(null=False, default=10, help_text="When multiple resolvers match, highest priority wins")
+    name = models.CharField(_("name"), max_length=255, unique=True, help_text=_("Descriptive name of this resolver"))
+    url = models.URLField(_("url"), help_text=_("Website for this authority (optional)"), blank=True, null=True)
```
Comment (actlikewill) [inline indigo_resolver/models.py:22]: maybe this needs to be capitalized?

File: indigo_resolver/models.py [hunk 2]
```diff
@@ -19,14 +18,15 @@ class Authority(models.Model):
...
+    not_found_url = models.URLField(_("not found url"), help_text=_("URL of a 404 page (optional)"), null=True, blank=True)
```
Comment (actlikewill) [inline indigo_resolver/models.py:23]: same here

File: indigo_resolver/models.py [hunk 3]
```diff
@@ -52,14 +52,14 @@ class AuthorityReference(models.Model):
...
-    updated_at = models.DateTimeField(auto_now=True)
-    priority = models.IntegerField(null=True, blank=True, help_text="Priority for this URL. If unset, uses the authority's priority")
-
-    authority = models.ForeignKey(Authority, related_name='references', on_delete=models.CASCADE)
+    frbr_uri = models.CharField(_("frbr uri"), max_length=255, db_index=True, help_text=_("FRBR Work or Expression URI to match on"))
+    title = models.CharField(_("frbr uri"), max_length=255, help_text=_("Document title"))
+    url = models.URLField(_("frbr uri"), max_length=1024, help_text=_("URL from which this document can be retrieved"))
+    created_at = models.DateTimeField(_("frbr uri"), auto_now_add=True)
+    updated_at = models.DateTimeField(_("frbr uri"), auto_now=True)
+    priority = models.IntegerField(_("frbr uri"), null=True, blank=True, help_text=_("Priority for this URL. If unset, uses the authority's priority"))
```
Comment (actlikewill) [inline indigo_resolver/models.py:60]: these seem to be incorrect

File: indigo_resolver/models.py (hunk 1 above)
Comment (longhotsummer) [reply indigo_resolver/models.py]: fixed

File: indigo_resolver/models.py (hunk 3 above)
Comment (longhotsummer) [reply indigo_resolver/models.py]: d'oh!

File: indigo_resolver/models.py (hunk 2 above)
Comment (longhotsummer) [reply indigo_resolver/models.py]: fixed

**Review by actlikewill — APPROVED** [review]
looks good

**Pull request merged**